cli/src/main/java/com/devonfw/tools/ide/environment/EnvironmentVariablesResolved.java

```diff
@@ -26,6 +26,11 @@ public EnvironmentVariablesType getType() {
     return EnvironmentVariablesType.RESOLVED;
   }
 
+  @Override
+  public void remove(String string) {
+    
```

Comment:
We might have to add an exception here as the whole implementation is missing.

Reply:
Added default implementation to EnvironmentVariables.

Reply:
BTW: Does it make sense to add a default implementation in the interface throwing an exception that says `Not yet implemented!`? It will IMHO never be implemented in `EnvironmentVariablesSystem` since it cannot work by design.

So if we want to cleanup here, remove it entirely from the interface and only add this method in `EnvironmentVariablesPropertiesFile`. That would be a reasonable design improvement to prevent users from calling this method in situations where it will not work.

Reply:
Thanks for elaborating, I've adjusted it now.

Reply:
Thanks. Maybe for `set` we should do the same thing, but for `remove` it is now solved hence closing this thread as resolved.